hooks/charmhelpers/contrib/openstack/neutron.py

```diff
@@ -204,7 +204,7 @@ def neutron_plugins():
                                         database=config('database'),
                                         ssl_dir=NEUTRON_CONF_DIR)],
             'services': [],
-            'packages': ['plumgrid-lxc',
+            'packages': ['plumgrid-edge-docker',
```

Comment:
For edge and gateway, this is the standard one but it will be clear when we also have the director docker package